src/test/java/features/Table Tests.feature

```diff
@@ -10,6 +10,7 @@ Feature: Table Tests
     When I click the Name header
     Then I verify the cells in the Name column in the Example Table are sorted in ascending order
       And I see 100 rows in the Example Table
+      And I verify the cell in the 1st row and the Name column of the Example table contains the text Alexander Foley
```

Comment:
These are good but I'd also like to make sure they work with the NGXDataTable. CAn you add the same tests to that feature file?